### PR TITLE
Bump version to 3.2

### PR DESCRIPTION
**Pull request by philippjfr**
Simple version bump.

### CODE REVIEW

**Review by Jrice1317 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bokeh-{{ version }}.tar.gz
-  sha256: ba0fc6bae4352d307541293256dee930a42d0acf92e760c72dc0e7397c3a28e9
+  sha256: cdb3a8c3109d8d91a2814f4cddea045c14d049021f7fb6ce2a6b2c15e224bb17
 
 build:
   number: 0
-  skip: true  # [py<38]
+  skip: true  # [py<39]
   script:
     - {{ PYTHON }} -m pip install . -vv
```
Comment (Arishamays1) [inline recipe/meta.yaml:16]: All good! My only suggestion is to perhaps amend the `script` to 
`script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv`

**Review by Arishamays1 — CHANGES_REQUESTED** [review]

**Review by ryanskeith — APPROVED** [review]
LGTM.

**Review by skupr-anaconda — APPROVED** [review]
LGTM but Arisha's suggestion should be addressed

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [inline recipe/meta.yaml:16]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

**Review by lorepirri — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: @Arishamays1 I've reformulated your proposed change as a GitHub `suggestion` and it was applied by @philippjfr

**Comment by lorepirri** [comment]
if you like:
`colorama 0.4.6`
`setuptools-git-versioning 1.13.1`

`doc_url: https://docs.bokeh.org/en/latest`

**Review by varlackc — APPROVED** [review]
LGTM

**Comment by philippjfr** [comment]
Resolved the remaining linter warnings, so this should be ready now.

**Review by lorepirri — APPROVED** [review]

**Comment by lorepirri** [comment]
I've deleted the branch and change the settings to do it automatically, plus protected the `master` branch with PRs and min 2 reviews.

**Pull request merged**